common/warp_utils/src/cors.rs

```diff
@@ -0,0 +1,21 @@
+use std::net::Ipv4Addr;
+use warp::filters::cors::Builder;
+
+/// Configure a `cors::Builder`.
+///
+/// If `allow_origin.is_none()` the `default_origin` is used.
+pub fn set_builder_origins(
+    builder: Builder,
+    allow_origin: Option<&str>,
+    default_origin: (Ipv4Addr, u16),
+) -> Builder {
+    if let Some(allow_origin) = allow_origin {
+        if allow_origin == "*" {
+            builder.allow_any_origin()
```

Comment:
This looks like it might set the header to null instead of * ? I'm not totally sure about the implications of that.. its does still work though.

Reply:
Hmm yeah, good point. I was pondering this over the weekend too. The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) state that we shouldn't use `null`.

As such, I've made a PR to `warp` to support the wildcard: https://github.com/seanmonstar/warp/pull/726. This commit switches us over to use the `github.com/paulhauner/warp` dependency.